pkg/option/option.go

```diff
@@ -46,6 +46,7 @@ type ClusterOptions struct {
 	StateFlag                string            `yaml:"state-flag"`
 	// Secondary members define URLs to connect to cluster formed by primary members.
 	PrimaryListenPeerURLs []string `yaml:"primary-listen-peer-urls"`
+	MaxSyncMessageSize    int      `yaml:"max-sync-message-size"`
```

Comment:
Why use a new name `MaxSyncMessageSize` and not the name `MaxCallSendMsgSize` in Etcd?
I think `MaxCallSendMsgSize` is from GRPC, and Etcd reuse it.

Reply:
Good point, we should probably reuse it also. I'll fix it